modules/ROOT/pages/rtf-deployment-settings.adoc

```diff
@@ -0,0 +1,73 @@
+= Configuring Runtime Fabric Deployment Settings
+
+You can deploy Partner Manager to an Anypoint Runtime Fabric configured with Amazon Elastic Kubernetes Service (EKS).
+
+== Prerequisites for Deploying to Runtime Fabric
+
+Before you deploy Partner Manager to Runtime Fabric, you must:
+
+* Ensure that you have the required permissions for using Runtime Fabric.
+* Create and configure the Runtime Fabric instance into which to deploy Partner Manager services, as described in https://developer.mulesoft.com/tutorials-and-howtos/runtime-fabric/runtime-fabric-aws-elastic-kubernetes-service[Getting Started with Runtime Fabric on Amazon Elastic Kubernetes Service (EKS)].
+* Configure Persistence Gateway in Runtime Fabric, as described in xref::docs-runtime-fabric:persistence-gateway.adoc[Persistence Gateway].
++
+Partner Manager uses a persistent Object Store via Persistence Gateway to store transaction processing information such as EDI control numbers and message routing information, which the Partner Manager runtime applications access.
+* Set up an ActiveMQ (JMS) messaging broker and configure the connection settings while configuring the Partner Manager Runtime Fabric deployment settings.
++
+Partner Manager uses ActiveMQ to reliably send payloads between different stages when it processes B2B transactions. The Partner Manager deployment service automatically creates the required message queues in your ActiveMQ instance when you deploy message flows.
+
+== Configure Settings for Deploying to Runtime Fabric
+
+To configure settings for deploying Partner Manager to Runtime Fabric, access the business unit and environment for which you want to deploy Partner Manager and then follow these steps:
+
+. In the *Configure a Partner Manager environment* window, enter the following general settings:
++
+[%header%autowidth.spread]
+|===
+| Setting | Description | Default 
+| Deployment Target Type | Target type for your Partner Manager deployment. Select *Runtime Fabric*. | CloudHub 1.0
+| Runtime Fabric | Name of the Runtime Fabric instance to use as your deployment target.  | 
+| Runtime Version | Mule runtime engine version to run your Runtime Fabric instance.| 4.4.0
+| Anypoint MQ Region | Region where Anypoint MQ is deployed.  | US East (N. Virginia)
```

Comment:
Please add the below note in the 2nd column.

When users replay transmissions in Partner Manager with runtime applications deployed to Runtime Fabric, Partner Manager's control plane sends the transmission identifiers that need to be replayed into an Anypoint MQ queue. The document processing service app running in your RTF environment reads the transmission id from the Anypoint MQ to initiate the reprocessing of the transmission within your runtime applications. No transaction payload is processed via Anypoint MQ.